ONSdigital/eq-questionnaire-launcher
# Make v2 default

### What is the context of this PR?
This is to change the version dropdown to default to v2. Just to make things a bit easier for the user.

- Also fixes console error around a null field trying to be set to null
- Tidies up some formatting
- Refactors the show hide css js for some elements to make more sense
- Fixes a bug with the default value for `ref_p_start_date` and `ref_p_end_date` not being used

### How to review
- Check that v2 is selected now when launcher is loaded
- Check that schemas load as expected when selected from dropdown and using a url
